src/Sylius/Bundle/UserBundle/Resources/config/routing/verification.yml

```diff
@@ -0,0 +1,14 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+sylius_user_request_verification_token:
+    path: /
+    methods: [POST]
+    defaults:
+        _controller: sylius.controller.user:requestVerificationTokenAction
+
+sylius_user_verification:
+    path: /{token}
+    methods: [GET]
```

Comment:
Not really accurate. GET should not update a database state, what actually happens in `verifyAction`...